Test failures due to NoClassDefFoundError

**Steps:**  Command used: ant test
**Failed Tests:** 107
**Stacktrace:**
java.lang.NoClassDefFoundError: com/google/common/base/Function
         at datafu.test.pig.PigTests.getJarPath(PigTests.java:152)
         at datafu.test.pig.PigTests.getDefaultArgs(PigTests.java:61)
         at datafu.test.pig.PigTests.getDefaultArgsAsList(PigTests.java:69)
         at datafu.test.pig.PigTests.createPigTest(PigTests.java:86)
         at datafu.test.pig.PigTests.createPigTestFromString(PigTests.java:80)
         at datafu.test.pig.stats.WilsonBinConfTests.wilsonTest(WilsonBinConfTests.java:52)
         at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
         at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
         at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
         at java.lang.reflect.Method.invoke(Method.java:606)
         at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
         at org.testng.internal.Invoker.invokeMethod(Invoker.java:691)
         at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:883)
         at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1208)
         at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
         at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
         at org.testng.TestRunner.privateRun(TestRunner.java:754)
         at org.testng.TestRunner.run(TestRunner.java:614)
         at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
         at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
         at org.testng.SuiteRunner.privateRun(SuiteRunner.java:292)
         at org.testng.SuiteRunner.run(SuiteRunner.java:241)
         at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
         at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
         at org.testng.TestNG.runSuitesSequentially(TestNG.java:1169)
         at org.testng.TestNG.runSuitesLocally(TestNG.java:1094)
         at org.testng.TestNG.run(TestNG.java:1006)
         at org.testng.TestNG.privateMain(TestNG.java:1316)
         at org.testng.TestNG.main(TestNG.java:1280)
     Caused by: java.lang.ClassNotFoundException: com.google.common.base.Function
         at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
         at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
         at java.security.AccessController.doPrivileged(Native Method)
         at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
		 at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
         at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
         at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
         ... 29 more

--- a/build.xml
+++ b/build.xml
@@ -225,6 +225,7 @@
     <fileset dir="${tools.lib.dir}">
       <include name="*.jar" />
     </fileset>
+    <fileset id="fs" dir="${packaged.lib.dir}" includes="${guava.jar}"/>
     <pathelement path="${pigtestclasses.dir}" />
   </path>
 

